src/main/asciidoc/_chapters/hbase-default.adoc

```diff
@@ -1949,6 +1949,34 @@ If the DFSClient configuration
 `simple`
 
 
+[[hbase.security.authentication.ui.metrics.protected]]
```

Comment:
We should add how to use this property and mention about its implication under security.adoc as well. See src/main/asciidoc/_chapters/security.adoc#13-using-spnego-for-kerberos-authentication-with-web-uis